taichi/ir/ir_builder.h

```diff
@@ -5,12 +5,13 @@
 TLANG_NAMESPACE_BEGIN
 
 class IRBuilder {
- private:
+ public:
   struct InsertPoint {
     Block *block;
     int position;
   };
 
+ private:
   std::unique_ptr<IRNode> root_;
```

Comment:
```suggestion
  std::unique_ptr<IRNode> root_{nullptr};
```

Reply:
This is actually set in the constructor... Do we need to initialize it here?

Reply:
Yeah, let's keep the style consistent